2025-04-13T09:14:47.2453286Z ##[group]Run source .venv/bin/activate
2025-04-13T09:14:47.2453632Z [36;1msource .venv/bin/activate[0m
2025-04-13T09:14:47.2453877Z [36;1mruff check .[0m
2025-04-13T09:14:47.2503094Z shell: /usr/bin/bash -e {0}
2025-04-13T09:14:47.2503337Z env:
2025-04-13T09:14:47.2503583Z   pythonLocation: /opt/hostedtoolcache/Python/3.13.2/x64
2025-04-13T09:14:47.2504009Z   PKG_CONFIG_PATH: /opt/hostedtoolcache/Python/3.13.2/x64/lib/pkgconfig
2025-04-13T09:14:47.2504414Z   Python_ROOT_DIR: /opt/hostedtoolcache/Python/3.13.2/x64
2025-04-13T09:14:47.2504779Z   Python2_ROOT_DIR: /opt/hostedtoolcache/Python/3.13.2/x64
2025-04-13T09:14:47.2505139Z   Python3_ROOT_DIR: /opt/hostedtoolcache/Python/3.13.2/x64
2025-04-13T09:14:47.2505502Z   LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/3.13.2/x64/lib
2025-04-13T09:14:47.2505792Z ##[endgroup]
2025-04-13T09:14:47.2981671Z ch05/01_main-chapter-code/exercise-solutions.ipynb:1:1: Expected a Jupyter Notebook, which must be internally stored as JSON, but this file isn't valid JSON: expected `,` or `]` at line 272 column 5
2025-04-13T09:14:47.2983064Z Found 1 error.
2025-04-13T09:14:47.3011188Z ##[error]Process completed with exit code 1.
